addons/l10n_din5008_expense/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'DIN 5008 - Expenses',
+    'category': 'Accounting/Localizations',
+    'depends': [
+        'l10n_din5008',
+        'hr_expense',
+    ],
+    'auto_install': True,
+    'license': 'LGPL-3',
+}
```

Comment:
nitpick: imo adding a `'version'` explicity makes sense (to make it easier to bump if necessary in the future)

Reply:
:warning: Not a nitpick